chart/RELEASE_NOTES.rst

```diff
@@ -23,6 +23,73 @@ Run ``helm repo update`` before upgrading the chart to the latest version.
 
 .. towncrier release notes start
 
+Airflow Helm Chart 1.12.0 (2024-01-22)
+--------------------------------------
+
+Significant Changes
+^^^^^^^^^^^^^^^^^^^
+
+The helm chart is now using a newer version of ``bitnami/postgresql`` dependency (#34817)
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+The version of ``bitnami/postgresql`` subchart upgraded from ``12.10.0`` to ``13.2.24``.
+The version of ``PostgreSQL`` binaries upgraded from ``11`` to ``16.1.0``.
+
+The change requires existing ``bitnami/postgresql`` subchart users to perform manual major version upgrade using ``pg_dumpall`` or ``pg_upgrade``.
+
+As a reminder, it is recommended to `set up an external database <https://airflow.apache.org/docs/helm-chart/stable/production-guide.html#database>`_ in production.
+
+Default Airflow image is updated to ``2.8.1`` (#36907)
+""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+The default Airflow image that is used with the Chart is now ``2.8.1``, previously it was ``2.7.1``.
+
+New Features
+^^^^^^^^^^^^
+
+- Enable native HPA for Airflow Workers (#36174)
+- Add init container + sidecar support for Airflow Kerberos (#35548)
+
+Improvements
+^^^^^^^^^^^^
+
+- Improve PriorityClass to improve debuggability (#36365)
```

Comment:
```suggestion
- Improve PriorityClass to improve debug capabilities (#36365)
```

With this it should not require adding the word to the spelling wordlist

Reply:
Is there an issue with adding the new word?

Reply:
just wondered if debuggability is a real word :)

Reply:
Oh, yeah, it is a real word :) not as common, but real.